dwa/src/components/LoginForm.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+interface LoginProps {
+    onLogin: (did: string, verifiableCredentials: string) => void;
+}
+
+const LoginForm = ({ onLogin }) => {
+    const [did, setDid] = useState('');
+    const [verifiableCredentials, setVerifiableCredentials] = useState('');
+
+    const handleDidChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setDid(e.target.value);
+    };
+
+    const handleVerifiableCredentialChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setVerifiableCredentials(e.target.value);
+    };
+
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        onLogin(did, verifiableCredentials);
+    };
+
+    return (
+        <form onSubmit={handleSubmit}>
+            <div>
+                <label htmlFor="did">Username:</label>
+                <input
+                    type="text"
+                    id="did"
+                    value={did}
+                    onChange={handleDidChange}
+                    required
+                />
+            </div>
+            <div>
+                <label htmlFor="verifiableCredentials">Password:</label>
+                <input
+                    type="verifiableCredentials"
```

Comment:
Afaik `verifiableCredentials` isn't an input type. This would just default to text, or is there something I'm missing decorating the input with this type? 

Also, we should be fetching the did from their dwn, they authenticate with the DID and grant us read access to the actual vc document data from the dwn. This text input isn't very useful